hangry_games/src/games.rs

```diff
@@ -48,7 +48,7 @@ impl Game {
     }
 
     pub fn run_day_night_cycle(&mut self) {
-        let mut game = get_game(self.name.as_str()).expect("Error loading game");
+        let game = get_game(self.name.as_str()).expect("Error loading game");
```

Comment:
Out of curiosity, do we remove the mutability here because it's no longer needed _now_, or did we not need it mutable initially either?

Reply:
IIRC, the updating of the day used to require the mutability.